test/factories/domain.rb

```diff
@@ -14,5 +14,10 @@
         FactoryBot.create(:domain_parameter, :domain => domain)
       end
     end
+
+    factory :domain_for_snapshots do
+      name { 'snapshotdomain.com' }
+      fullname { 'snapshotdomain.com' }
```

Comment:
Should we use `example.com`? or `snap.example.com`

Reply:
I wanted to differentiate this from exampleN.com but sure.